frontend/src/app/admin/associations/edit-association/edit-association.component.ts

```diff
@@ -0,0 +1,75 @@
+import { Observable } from 'rxjs';
+
+import { CommonModule } from '@angular/common';
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, FormsModule, ReactiveFormsModule, Validators } from '@angular/forms';
+import { ActivatedRoute, RouterModule } from '@angular/router';
+
+import { LetDirective } from '@app/common/directives/let/let.directive';
+import { FormErrorMessagePipe } from '@app/common/pipes/form-error-message/form-error-message.pipe';
+
+import { CreateAssociationFormGroup } from '../create-association/common/models/create-association.model';
+import { CreateAssociationState } from '../create-association/create-association.store';
+import { EditAssociationStore } from './edit-association.store';
+
+@Component({
+    selector: 'app-edit-association',
+    standalone: true,
+    imports: [CommonModule, FormsModule, ReactiveFormsModule, FormErrorMessagePipe, RouterModule, LetDirective],
+    providers: [EditAssociationStore],
+    templateUrl: './edit-association.component.html'
+})
+export class EditAssociationComponent implements OnInit {
+    id?: number;
+    form!: FormGroup<CreateAssociationFormGroup>;
+    profile$ = this.editAssociationStore.profile$;
+    vm$: Observable<CreateAssociationState> = this.editAssociationStore.vm$;
+
+    selectedFile: File | null = null;
```

Comment:
I think we don't need it, maybe take a look at the user creation form to check how the image upload works.